.github/workflows/ci.yml

```diff
@@ -47,7 +47,8 @@ jobs:
         uses: actions/cache@v3.3.2
         with:
           path: ${{ steps.setup-haskell.outputs.cabal-store }}
-          key: ${{ runner.os }}-${{ matrix.ghc }}-${{ hashFiles('cabal.project.freeze') }}
+          key: ${{ runner.os }}-ghc-${{ env.GHC_VER }}-cabal-${{ env.CABAL_VER }}-${{ hashFiles('**/plan.json') }}
+          restore-keys: ${{ runner.os }}-ghc-${{ env.GHC_VER }}-cabal-${{ env.CABAL_VER }}-
```

Comment:
`env.GHC_VER` and `env.CABAL_VER` are likely empty here...

Reply:
Apologies, should be better now